### ISSUE TITLE
Crash with Architecturecraft-ElytraDev Version

### ISSUE THREAD

**Issue by Jorgee9001** [darkevilmac/archimedes-ships-plus#258]
Davinci's Vessels Version: 1.12-6.340-full
MovingWorld Version: 1.12-6.342-full
Minecraft 1.12.2
Forge version 14.23.5.2772

Crash report: https://paste.dimdev.org/adozakewiw.mccrash

Upon placing a structure from architecturecraft (in this case, a railing for a staircase) and assembling the ship, game crashes. The structure is fine on the ship until it's assembled.

Edit: This should be the right crash report.

**Comment by darkevilmac** [comment]
I think you've got the wrong crash report.

**Comment by Jorgee9001** [comment]
You're right... the report is completely different. I'm not sure why it didn't link the right one. I'll try to reproduce and edit it in.
